backend/dal/Helpers/Extensions/ServiceExtensions.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.EntityFrameworkCore;
+using Pims.Dal.Exceptions;
+using Pims.Dal.Security;
+using Pims.Dal.Services;
+using System;
+using System.Linq;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Dal.Helpers.Extensions
+{
+    /// <summary>
+    /// ServiceExtensions static class, provides extension methods for services.
+    /// </summary>
+    public static class ServiceExtensions
+    {
+        #region Variables
+        public static readonly string[] EDITABLE_STATUS_CODES = new[] { "DR", "DR-P", "DR-I", "DR-D", "DR-A" };
```

Comment:
I don't love this, but for now it'll have to do.  We use `Codes` for permanent values.  Which allows for flexibility when initializing or change databases.